include/z64item.h

```diff
@@ -257,16 +257,16 @@ typedef enum {
     /* 0x6D */ ITEM_GORON_RUBY,
     /* 0x6E */ ITEM_ZORA_SAPPHIRE,
     /* 0x6F */ ITEM_STONE_OF_AGONY,
-    /* 0x70 */ ITEM_GERUDO_CARD,
+    /* 0x70 */ ITEM_GERUDOS_CARD,
```

Comment:
Looks like `QUEST_GERUDO_CARD` should also be updated, and possibly other quest enum values